dags/tests/test_dag_integrity.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+import pytest
+from airflow import models as airflow_models
+
+DAG_DIR = Path(__file__).parent.parent
+ALL_PATHS = set(DAG_DIR.rglob('*.py'))
+EXCLUDED_PATHS = (
+    set(DAG_DIR.rglob('_*.py'))
+    .union(set((DAG_DIR / 'tests').rglob('*.py')))
+    .union({DAG_DIR / 'airflow_utils.py', DAG_DIR / 'kube_secrets.py'})
+)
+
+DAG_PATHS = ALL_PATHS - EXCLUDED_PATHS
+
+
+@pytest.mark.parametrize("dag_path", [pytest.param(path, id=path.name) for path in DAG_PATHS])
+def test_dag_integrity(dag_path):
+    """Import DAG files and check for a valid DAG instance."""
+    dag_name = dag_path.name
+    module = _import_file(dag_name, dag_path)
```

Comment:
0/5, but since you don't use `dag_name` anywhere else, you could've extracted it inside `_import_file`.